plugins/home/src/homePageComponents/FeaturedDocsCard/Content.tsx

```diff
@@ -116,7 +116,7 @@ export const Content = (props: FeaturedDocsCardProps): JSX.Element => {
               }/`
             }
           >
-            {d.metadata.title}
+            {d.metadata.title ?? d.metadata.name}
```

Comment:
```suggestion
            <EntityDisplayName entityRef={stringifyEntityRef(d)} />
```

I think we'd want to update this to use the new `<EntityDisplayName/>` component with the entity ref. It was created for exactly this case, see https://github.com/backstage/backstage/issues/20955

Reply:
Indeed this seems like a perfect use case for that.

Reply:
My understand this would add additional API calls for each element? 

But I will change it as it is definitely a cleaner implementation.

Reply:
It does, but with efficient batching and debouncing. We're trying to standardize on those components to ensure that the presentation is uniform across the interface